### PR TITLE
Add homepage and all necessary linters files

### PR DESCRIPTION
**Pull request by jrosario19**
## Added the home page in mobile version:
- Navbar.
- Headline section

### I also include the files needed in the project.
- Index.html file, Include the Nav 
- style.css file
- All necessary linters files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <title>Home Page</title>
+</head>
+<body>
+    <header>
+        <nav class="toolbar">
+            <ul class="navbar-items">
+                <li class="logo"> <h3>Welcome</h3></li>
```
Comment (keroloussamy) [inline index.html:15]: - [ ] Kindly wrap your `<h3>Welcome</h3>` inside an `a` tag, to make it link to your home page, every logo should link to the home page.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  font-size: 40px;
+  margin: 0 24px;
+}
+
+.headline span {
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  color: #36b37e;
+  margin: 0 24px;
+}
```
Comment (keroloussamy) [inline styles.css:54]: - [ ] Kindly remove the `margin: 0 24px;` role, to match the figma design.

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            <h1>Hey there. I’m Juan Fco. <br> <span>I’m a software developer</span></h1>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <ul class="contact-info">
+                <li> <a href="https://twitter.com/jrosario19" class="contact-info-button"><img src="images/twitter.svg" alt=""></a> </li>
+                <li> <a href="https://linkedin.com/in/juan-francisco-rosario-suli-44595051" class="contact-info-button"><img src="images/Linkedinicon.svg" alt=""></a> </li>
+                <li> <a href="#" class="contact-info-button"><img src="images/medium.svg" alt=""></a> </li>
+                <li> <a href="https://github.com/jrosario19" class="contact-info-button"><img src="images/github.svg" alt=""></a> </li>
+                <li> <a href="#" class="contact-info-button"><img src="images/angellist.svg" alt=""></a> </li>
+            </ul>
```
Comment (keroloussamy) [inline index.html:31]: - [ ] Kindly add this button at the end of your headline section, to match the Figma design.

This one
image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @jrosario19, 

Good job so far! You really did a great work
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <title>Home Page</title>
+</head>
+<body>
+    <header>
+        <nav class="toolbar">
+            <ul class="navbar-items">
+                <li class="logo"> <a href="index.html"><h3>Welcome</h3></a></li>
```
Comment (smunozmo) [inline index.html:15]: - Good work using an `a` tag on your logo.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  background-image: url("images/header-illsutration-mobile@2x.jpg");
+  background-size: cover;
+  width: 100%;
+  height: 100vh;
+}
+
+.headline h1 {
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+}
```
Comment (smunozmo) [inline styles.css:50]: - The margin was removed, well done!

File: index.html [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <ul class="contact-info">
+                <li> <a href="https://twitter.com/jrosario19" class="contact-info-button"><img src="images/twitter.svg" alt=""></a> </li>
+                <li> <a href="https://linkedin.com/in/juan-francisco-rosario-suli-44595051" class="contact-info-button"><img src="images/Linkedinicon.svg" alt=""></a> </li>
+                <li> <a href="#" class="contact-info-button"><img src="images/medium.svg" alt=""></a> </li>
+                <li> <a href="https://github.com/jrosario19" class="contact-info-button"><img src="images/github.svg" alt=""></a> </li>
+                <li> <a href="#" class="contact-info-button"><img src="images/angellist.svg" alt=""></a> </li>
+            </ul>   
+            <a href="#"><img src="images/Disabled.svg" alt=""></a>
```
Comment (smunozmo) [inline index.html:32]: - You added the missing button, great job.

**Review by smunozmo — APPROVED** [review]
Hi @jrosario19 , 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**